connector/src/main/java/org/geysermc/connector/entity/Entity.java

```diff
@@ -196,10 +201,14 @@ public void moveAbsolute(GeyserSession session, Vector3f position, Vector3f rota
         moveEntityPacket.setTeleported(teleported);
 
         session.sendUpstreamPacket(moveEntityPacket);
+
+        if (positionChanged) {
+            checkPlayerCollision(session);
+        }
     }
 
     /**
-     * Teleports an entity to a new location. Used in JavaEntityTeleportTranslator.
+     * Teleports an entity to a new location. Used in {@link org.geysermc.connector.network.translators.java.entity.JavaEntityTeleportTranslator}.
```

Comment:
I would just import these, it’s much cleaner.

Reply:
For Javadocs?

Reply:
Yes, intellij should give you a context action to do if